example-apps/chatbot-rag-app/env.example

```diff
@@ -35,7 +35,7 @@ ES_INDEX_CHAT_HISTORY=workplace-app-docs-chat-history
 # VERTEX_REGION=
 # GOOGLE_APPLICATION_CREDENTIALS=
 
-# Uncomment and complete if you want to use Mistral AI
+# Uncomment and complete if you want to use Mistral AI Large depployed through Microsoft Azure
```

Comment:
I don't understand this. Nothing in this change has a constraint on a specific endpoint or model, so why mention that here?

Reply:
That's a good point. I have yet to test the code against a direct Mistral AI endpoint. The mention was there to clarify that this was tested with the Mistral AI Large deployed on Azure in case there is anything different about direct Mistral AI endpoints. We can change it back.

Reply:
If we want to make a disclaimer that this was tested only on a specific subset, I'd suggest doing so in the readme file, where the documentation is, maybe even include deployment instructions and example values for the endpoint and model environment variables.

The code change is really not dependent on a specific deployment method or model though, so I suggest being more specific, like for example adding a note along the lines of "While all Mistral AI models should be supported, we have only tested the Mistral AI Large model deployed on Azure".

Reply:
Thanks, Miguel. I will update the Readme